docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
```

Comment:
@newPOPE If I merge this pull request would you volunteer to send a new PR changing it to use the official image?